docker/intel-dgpu-driver.Dockerfile

```diff
@@ -33,4 +33,4 @@ COPY --from=builder /etc/driver-toolkit-release.json /etc/
 COPY --from=builder /lib/modules/$KERNEL_FULL_VERSION/ /opt/lib/modules/$KERNEL_FULL_VERSION/
 COPY --from=builder /build/intel-gpu-firmware/firmware/ /firmware/i915/
 
-RUN depmod -b /opt
```

Comment:
why we have this line? :-)

Reply:
That line runs depmod from the /opt directory in the final driver container image. It creates the modules.dep file that creates a dependency list for each driver. For example, i915 requires intel_vsec to be loaded first. I believe this is required. During the build, a depmod is run but it is in the default /lib directory. We had added this line based on the KMM examples: https://openshift-kmm.netlify.app/lab/#in-cluster-build-from-sources-single-step.

Reply:
COPY --from=builder /etc/driver-toolkit-release.json /etc/: This line is optional, it simply adds the driver-toolkit version in a file for reference. We can remove it in a separate PR. It was something we inherited from early SRO/KMM examples.

Reply:
Sorry, I didn't express my meaning clearly. 
The line is right.
But I'd like to know why the previous patch, it removes the line and then add the same line?
Maybe the two lines have some characters that can't be printed like space?

thanks!

Reply:
![image](https://user-images.githubusercontent.com/82060795/236871644-905876a5-14e2-4ea7-8bd8-562c1de7785b.png)

Reply:
There was no newline at the end of line 36 before.